src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -107,9 +108,11 @@ public PersistentDataStore() {
         String title = (String) entity.getProperty("title");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
         Boolean isPrivate = (Boolean) entity.getProperty("is_private");
+        List<String> membersAsList = (List<String>) entity.getProperty("members");
```

Comment:
Is it possible to directly cast as a HashSet?

Reply:
No unfortunately.